src/drivers/intel/cavs/hda.c

```diff
@@ -49,4 +157,9 @@ const struct dai_driver hda_driver = {
 		.probe			= hda_dummy,
 		.remove			= hda_dummy,
 	},
+	.ts_ops = {
+		.ts_config		= hda_ts_config,
+		.ts_start		= hda_ts_start,
+		.ts_get			= hda_ts_get,
```

Comment:
Is there also a ts_stop?

Reply:
The cAVS feature does not need a stop but it could be useful generally if there would be more to set up. The DMA LLPs are enabled now by default but it they weren't then start/stop would need to control them.

I feel it could be useful to add stop as preparation for future or for other platforms. What do you think @lgirdwood @dbaluta ?

Reply:
Done. I've added stop into the latest version. As suggested by Liam I should add note the it's optional.